translations/messages+intl-icu.de.yml

```diff
@@ -3969,6 +3970,7 @@ warning.isolated.tag: "Der Stellungnahme {parentStatementExternalId} ist (weiter
 warning.limited.input.field: "Das Eingabefeld  \"{fieldname}\" ist auf {limit} Zeichen begrenzt."
 warning.local.participant.confirm: "Bitte bestätigen Sie, dass Sie in der/einer Gemeinde wohnen, die dieses Thema betrifft."
 warning.login.failed: "Der Login war nicht erfolgreich. Bitte versuchen Sie es erneut."
+warning.login.failed.throttle: "Zu viele erfolglose Loginversuche. Bitte versuchen Sie es in {minutes} Minute erneut."
```

Comment:
```suggestion
warning.login.failed.throttle: "Sie haben sich zu oft erfolglos eingeloggt. Bitte versuchen Sie es in {minutes} Minuten erneut."
```

Reply:
Thank you 5a530824Atm it is always one minute and I wanted to avoid the complexity of dynamic transkeys

Reply:
Minior: the wording "erfolglos eingeloggt" wihtout the term of "trying" doesn't make sense to me. But honestly I do not have a suggestion.

Reply:
Not sure whether I understand you correctly. The user just actively tried to log in and receives this message in a notification box. The context of the action should be clear to the user imho